### DISCUSSION TITLE
link

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#10] [Announcements]
# link

chaofa 的友情链接, 这里记录 chaofa 的一些朋友们

https://bruceyuan.com/link.html

**Comment by OnlyPiglet** [comment]
站点名称: 代码笔记
站点描述: 热爱生活，保持好奇心
站点地址: https://freelifeblog.top/
已加友链：https://freelifeblog.top/link.html
站点Icon: https://freelifeblog.top/icons-192.png

**Reply by bbruceyuan** [reply]
已添加~ 原来一直没回复你是因为「我的博客本来也没什么搜索量」，闹着玩的，也不打算正经维护 link，所以不太关注这个
